### ISSUE TITLE
Medium Montage

### ISSUE THREAD

**Issue by hamid13r** [eisfabian/SPACEtomo#2]
Dear Fabian,

After getting the whole grid montage the code identifies the lams with high accuracy. It does move to the lams and does the eccentric height routine, but when it goes for collecting the montage I get blank images. It doesn't look like the blank image when there is no beam, I checked and I have the beam and I am on the target. I also trying view mag and a polygon and it looks normal. Anything that could be going wrong in the `collectMMMap` function?

<img width="387" alt="Screenshot 2024-01-28 at 8 57 58 PM" src="https://github.com/eisfabian/SPACEtomo/assets/66981513/79c464b8-0e2f-4a60-ac79-09d687e2017f">

**Comment by eisfabian** [comment]
What's the log output during the MMMap collection?
It should tell you how many tiles it tries to collect.

The picture you posted is the map inside SerialEM? Or the one saved as a png file?

**Comment by hamid13r** [comment]
I am sorry I seem to have overwritten the log output. But I have a screenshot that seems to have the log output in it. 

Here are the screenshots and the last one is the png file saved after the montage was done. 

![Capture](https://github.com/eisfabian/SPACEtomo/assets/66981513/3928b86b-597c-466a-a8d7-88bee5dd99e0)
![Screenshot 2024-02-05 at 12 42 53 PM](https://github.com/eisfabian/SPACEtomo/assets/66981513/47439974-bb0b-4ec5-b580-cd36816ceeba)
![MIM056_3_L01](https://github.com/eisfabian/SPACEtomo/assets/66981513/12f12cbe-f809-43ea-922c-3d749a371a0b)

**Comment by eisfabian** [comment]
Thank you! The number tiles (2x4) looks reasonable.

The eucentric height routine ran on the lamella?

It looks like it's the same tile distorted over and over.
What does the montage look like when you open the mrc file in 3dmod? 

What overlap did you set? (MM_montage_overlap)

**Comment by hamid13r** [comment]
Sorry for the late reply, I had to go back to krios to get some info. The eucentric runs fine everytime. 
[MIM056_1-9.log](https://github.com/eisfabian/SPACEtomo/files/14270514/MIM056_1-9.log)

the mrc file looks like this, I noticed it is using the search preset this time. 
![image](https://github.com/eisfabian/SPACEtomo/assets/66981513/03c4b631-b495-4456-8c27-e2a90f9f1dd5)

overlap was set to 0.25. I am attaching all the settings:
[spacetomo_settings.txt](https://github.com/eisfabian/SPACEtomo/files/14270570/spacetomo_settings.txt)

**Comment by eisfabian** [comment]
This is really weird. Everything I can think off looks good.
You said, the images are already blank during the collection?

Can you try running these commands in a small script? You can also play around with the parameters a bit and see if you get it to work.

```
OpenNewMontage 2 3 test.mrc
SetMontageParams 1  500 500 4000 4000 
# Parameters: stage move, overlap X, overlap Y, piece size X, piece size Y, skip correlation, binning
M
```

Thanks and best wishes,
Fabian

**Comment by eisfabian** [comment]
This should be fixed in version 1.1.
